plugins/plugin-0x/src/actions/swap.ts

```diff
@@ -110,7 +111,7 @@ export const swap: Action = {
 					text: `✅ Swap executed successfully!\nView on Explorer: ${CHAIN_EXPLORERS[chainId]}/tx/${txHash}`,
 					content: { hash: txHash, status: "success" },
 				});
-				return true;
+				return traceResult(state, true);
```

Comment:
Remember for each of the trace results, we should actually be returning the in this example the output like the transaction hash in the next example with the charge the charge like we should be returning the actual like API call output you know if there is any